polars/io.ts

```diff
@@ -1,13 +1,36 @@
-import {DataType} from "./datatypes";
+import { DataType } from "./datatypes";
 import pli from "./internals/polars_internal";
-import {DataFrame, _DataFrame} from "./dataframe";
-import {isPath} from "./utils";
-import {LazyDataFrame, _LazyDataFrame} from "./lazy/dataframe";
-import {Readable, Stream} from "stream";
-import {concat} from "./functions";
-
+import { DataFrame, _DataFrame } from "./dataframe";
+import { isPath } from "./utils";
+import { LazyDataFrame, _LazyDataFrame } from "./lazy/dataframe";
+import { Readable, Stream } from "stream";
+import { concat } from "./functions";
+
+export interface readCsvOptions {
```

Comment:
can we make these interfaces `PascalCase`.

Reply:
Done!